src/Controller/PageController.php

```diff
@@ -99,6 +100,23 @@ public function page(string $slug, Request $request)
             $breadcrumb[$link] = $name;
         }
 
+        // request to option cards api
+        $optionCardsUrl = getenv('APP_API_BASE_URL') . 'ccs/v1/option-cards/0';
+        $optionCardsContent = null;
+
+        $client = HttpClient::create();
+        $response = $client->request(
+            'GET',
+            $optionCardsUrl,
+        );
+
+        if ($response->getStatusCode() == 200) {
+            $optionCardsContent = json_decode($response->getContent());
+            // dd($ctaContent);
+        } else {
+            $optionCardsContent = null;
+        }
```

Comment:
Seems redundant after the initialisation above on line 105. I would move the initialisation to just before the condition.

Reply:
Ah yep, that makes sense. Thanks.